src/main/java/com/qaprosoft/carina/demo/gui/pages/ModelInfoPage.java

```diff
@@ -24,6 +24,8 @@
 public class ModelInfoPage extends AbstractPage {
     @FindBy(css = ".help-display strong")
     private ExtendedWebElement displayInfoLabel;
+    
+    private static final String s2 = null;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS00115)